openmmml/models/macepotential.py

```diff
@@ -52,13 +52,13 @@ class MACEPotentialImpl(MLPotentialImpl):
     To use one of the pre-trained MACE-OFF23 models, specify the model name. For example:
     >>> potential = MLPotential('mace-off23-small')
 
-    Other available MACE-OFF23 models include `mace-off23-medium` and `mace-off23-large`.
+    Other available MACE-OFF23 models include 'mace-off23-medium' and 'mace-off23-large'.
 
     To use a locally trained MACE model, provide the path to the model file. For example:
     >>> potential = MLPotential('mace', modelPath='MACE.model')
 
     During system creation, you can optionally specify the precision of the model using the
-    `precision` keyword argument. Supported options are 'single' and 'double'. For example:
+    precision` keyword argument. Supported options are 'single' and 'double'. For example:
```

Comment:
You accidentally removed the quote at the start of the line.

Reply:
Ups! Fixed.